package.json

```diff
@@ -77,9 +77,9 @@
         "publish_package": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run publish_package $2 ;}; f",
         "serve": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run serve $npm_config_open;}; f",
         "start": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run start ;}; f",
-        "test": "f () { bash ./scripts/circleci-config.test.sh && npm run test:stylelint  && npm run test:eslint-all && JEST_MAX_WORKERS=6 npm run test:jest  ;}; f",
+        "test": "f () { bash ./scripts/circleci-config.test.sh && npm run test:stylelint  && npm run test:eslint-all && npm run test:jest  ;}; f",
```

Comment:
The naming of "circleci" ought to be deprecated.

Reply:
the reason i didn't touch circleci config in this pr is that, i think its better to have both of the configs at first, then test and compare. then if everything was ok, i remove the circle ci config in another PR.